src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx

```diff
@@ -120,7 +120,11 @@ export function GroupContainer({
   };
 
   const setEditIndex = (index: number) => {
-    dispatch(FormLayoutActions.updateRepeatingGroupsEditIndex({ group: id, index }));
+    // if edit button has been clicked while edit container is open, we trigger validations if present in triggers
+    const validate: boolean = (index === -1) && container.triggers?.includes(Triggers.Validation);
```

Comment:
Though you type it a boolean in typescript it isn't necessarily so...
If `container.triggers` is not defined the `validate` will be undefined.
How about this instead:
```js
const validate = (index === -1) && !!container.triggers?.includes(Triggers.Validation);
```

Reply:
I guess this makes more sense. Validate will only run when passing `true`, so undefined is totally fine. But I like your suggestion, better to be explicit 👍

Reply:
I mean if you typed the variable, you might as well stick to the type :)